### ISSUE TITLE
Accessibility: Unlabeled buttons on album playlist

### ISSUE THREAD

**Issue by benpate** [EmissarySocial/bandwagon#136]
https://caneandable.social/@WeirdWriter/114091728016131227

**Comment by rkingett** [comment]
The buttons above the album list appear to be unlabeled as well, but ARIA can be hit and miss on mobile so best to stick with native labels if at all possible!

**Comment by benpate** [comment]
I've made a number of improvements on the album and track pages.  This is still FAR from perfect, but I this should be is a big step forward.

I also have a few more accessibility tools that I can use to make Bandwagon work better on screen readers, so the next round of updates will be easier.

**Comment by rkingett** [comment]
I checked one album and the buttons appear to be labeled so I think this can be closed! Fantastic work!

**Comment by benpate** [comment]
Glad this is working.

I'm sorry this wasn't clear from my note.  I've moved this into "Ready to Release" which means that I've completed additional changes that aren't live yet. I'm expecting to post the next update this weekend, which should be even better for screen readers.

Now that I have VoiceOver working, I'm looking forward to making a lot more accessibility improvements.

**Comment by benpate** [comment]
I posted more big updates this evening. Hopefully this makes the accessibility even better. Let me know how it’s working for you?
